src/plone/recipe/zope2instance/__init__.py

```diff
@@ -91,20 +90,12 @@ def install(self, update=False):
             if os.path.exists(location):
                 shutil.rmtree(location)
 
-            # Which Zope2 version do we have?
-            dist = pkg_resources.get_distribution('Zope2')
-            parsed = dist.parsed_version
-            major, minor = parsed[0:2]
-            major = int(major)
-            # We support creating instances for 2.12, 2.13 and 4.0
-            if major >= 4:
-                major = 2
-                minor = 13
-            elif major == 2:
-                minor = int(minor)
-                if minor > 13:
-                    minor = 13
-            version = str(major) + str(minor)
+            # Which Zope version do we have?
+            # dist = pkg_resources.get_distribution('Zope')
+            # parsed = dist.parsed_version
+            # We support creating instances for 4 only,
+            # which has the same structure as 2.13.
+            version = '213'
```

Comment:
isn't this very very confusing?

Reply:
The reason I kept the version 213 is because it meant I did not need to rename the skeleton directory, which would make `git blame` and friends harder.

Reply:
Afaik a "git mv oldname newname" keeps the history.

Reply:
Sure. Okay, I have renamed the directory to `skel4`. Indeed less confusing.